extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js

```diff
@@ -88,7 +88,7 @@ export class AmpFlyingCarpet extends AMP.BaseElement {
     clip.appendChild(container);
     this.element.appendChild(clip);
 
-    this.getViewport().addToFixedLayer(container);
+    this.getViewport().addToFixedLayer(container, false);
```

Comment:
Add a comment here explaining we want the carpet content have the viewer header offset but _not_ reparented to the fixed layer.